qiskit-superstaq/qiskit_superstaq/superstaq_provider.py

```diff
@@ -305,7 +304,7 @@ def cq_compile(
             ValueError: If `target` is not a valid CQ target.
         """
         if not target.startswith("cq_"):
-            raise ValueError(f"{target} is not a CQ target")
+            raise ValueError(f"{target!r} is not a valid CQ target.")
```

Comment:
Reason for !r in string formatting here vs. elsewhere?

Reply:
i tried to do this everywhere a target appears in an error message - imo it's more clear because it clearly shows the target string as it was passed by the user, e.g.
```
ValueError: 'ibmq_qasm_simulator' is not a valid CQ target.
```
instead of just
```
ValueError: ibmq_qasm_simulator is not a valid CQ target.
```